netbout-web/pom.xml

```diff
@@ -736,6 +736,11 @@
                                 <exclude>checkstyle:/src/main/scss/.*</exclude>
                                 <exclude>checkstyle:/src/test/casperjs/.*</exclude>
                                 <exclude>checkstyle:/src/main/js/.*</exclude>
+                                <!--
+                                @todo #911 Fix findbugs errors in web
+                                 filters in qulice 0.12 are broken, right excludes
+                                 should be more granular
+                                -->
                                 <exclude>findbugs:.*</exclude>
```

Comment:
@exper0 btw I would suggest you create a ticket in Qulice project
perhaps it is a bug 
If they accept the bug you can put link to it here